### DISCUSSION TITLE
is github support pure ssh for git-lfs protocol?

### DISCUSSION THREAD

**Discussion by fadhil-riyanto** [community/community#131493] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

hi everyone, few days ago, I was ask to git-lfs maintainer about pushing file using git-lfs, you can see issue [here](https://github.com/git-lfs/git-lfs/issues/5820), he said "GitHub doesn't yet support the pure SSH"

so my question, Is there have a comprehensive tutorial about uploading files to github using git-lfs?

**Comment by carlahall678** [comment]
GitHub does not support the Git Large File Storage (Git LFS) protocol over pure SSH. While Git itself can be accessed over SSH, Git LFS operations are typically handled over HTTPS. This means that even [click here](https://apkyup.com.tr/)  if you are using SSH for normal Git operations (like cloning, pushing, and pulling repositories), Git LFS will still require HTTPS for transferring large files.

**Comment by tshareef4563** [comment]
As of the latest updates, GitHub does not support pure SSH for the Git Large File Storage (Git LFS) protocol. Git LFS primarily relies on HTTPS for transferring large files. Even when you clone a  [visit website](https://multanrealtors.pk/plots-for-sale-in-multan/) repository using SSH, the actual transfer of LFS files still uses HTTPS behind the scenes.
